chainio/README.md

```diff
@@ -0,0 +1 @@
+# Chainio 
```

Comment:
nit: remove file or add some basic info?
Your diagram from the PR body would fit here nicely I think. And the mermaid syntax should be supported too.

Reply:
Will do!